cilium-health/launch/endpoint.go

```diff
@@ -107,17 +108,15 @@ func configureHealthRouting(netns, dev string, addressing *models.NodeAddressing
 	return err
 }
 
-// Client wraps a client to a specific cilium-health endpoint instance, to
-// provide convenience methods such as PingEndpoint().
+// Client wraps a probe client
```

Comment:
nit: The rest of the old comment is still valid, the new comment seems entirely pointless to me (the code immediately below already tells this very succinctly :-) )

Reply:
I fully agree! Not even sure why I edited that comment :)